docs/Community/SecurityAdvisoryHandling.md

```diff
@@ -0,0 +1,81 @@
+# Security Advisory Process for Maintainers
+
+## Reporting
+
+For instructions on reporting a vulnerability, refer to the Open Enclave SDK [SECURITY.md](https://github.com/openenclave/openenclave/security/policy#reporting-a-vulnerability).
+
+Vulnerability reporting for the OE SDK is currently still being handled through the [Microsoft Security Response Center (MSRC)](https://www.microsoft.com/en-us/msrc) as they:
+
+- provide a 24-hour turnaround time for acknowledging reports
+- maintain a [PGP key](https://www.microsoft.com/en-us/msrc/pgp-key-msrc#:~:text=MSRC%20PGP%20Key%20Last%20updated%3A%20October%2031%2C%202019,us.%20Click%20here%20to%20submit%20a%20security%20vulnerability) for secure reporting of vulnerabilities to secure@microsoft.com.
+
+Once a vulnerability related to the Open Enclave SDK is reported to MSRC, the report is forwarded to the committers responsible for reviewing security issues:
+
+- Anand Krishnamoorthi (@anakrish)
+- Ming-wei Shih (@mingweishih)
+- Simon Leet (@codemonkeyleet)
+
+> - [ ] TODO: Formalize SIG-Security and its membership with the CGC.
+
+## Handling Embargoed Vulnerabilities
+
+The security reviewers are responsible for the initial triage of the report. This includes:
+
+- reaching out to the appropriate OE SDK area owners or subject matter experts
+- evaluating the severity of the issue against the bug bar
```

Comment:
evaluating the severity and exploitability of the issue

TODO: OE should adopt baseline guidance of how OE vulnerabilities might map to Critical, Important, Moderate, and Low severity.

Reply:
This is the same TODO as indicated on line 29.